Kickstarter-iOS/Features/ReportProject/ReportThisProjectListItem.swift

```diff
@@ -0,0 +1,90 @@
+import Foundation
+import Library
+
+enum ReportProjectInfoListItemType {
+  case child
+  case parent
+}
+
+struct ReportProjectInfoListItem: Identifiable {
+  var type: ReportProjectInfoListItemType
+  var id = UUID()
+  var title: String
+  var subtitle: String
+  var subItems: [ReportProjectInfoListItem]?
+  var exampleMenuItems: [ReportProjectInfoListItem]?
+}
+
+/// Hierarchical structure of the ReportProjectInfo List View
+let listItems =
+  [
+    ReportProjectInfoListItem(
+      type: .parent,
+      title: Strings.This_project_breaks(),
+      subtitle: Strings
+        .Projects_may_not_offer(prohibited_items: "https://www.kickstarter.com/rules/prohibited?ref=project-page-report"),
+      subItems: thisProjecBreaksSubListItems
+    ),
+    ReportProjectInfoListItem(
+      type: .parent,
+      title: Strings.Report_spam(),
+      subtitle: Strings
+        .Our(community_guidelines: "https://www.kickstarter.com/help/community?ref=project-page-report"),
```

Comment:
Do we have these URLs somewhere in the app already? I wasn't seeing them and I'd rather not hard code these if we can avoid it.